tests/runtime/conftest.py

```diff
@@ -246,13 +247,12 @@ def register_trained_model(
     the old auto-load feature which was only needed for unit tests
     """
     model_future = MODEL_MANAGER.get_model_future(training_id)
-    model = model_future.load()
     loaded_model = (
         LoadedModel.Builder()
         .id(model_id)
         .type("trained")
         .path("")
-        .module(model)
```

Comment:
There are a lot of places where the words `module` and `model` are intermixed. In this case, `module` felt distinctly incorrect because `module` should be the `class` while `model` should be the `instance` and the _whole_ point here is to create an `instance`.